docs/topics/migrations.txt

```diff
@@ -652,6 +652,7 @@ Django can serialize the following:
 - ``datetime.date``, ``datetime.time``, and ``datetime.datetime`` instances
   (include those that are timezone-aware)
 - ``decimal.Decimal`` instances
+- ``functools.partial`` instances which have serializable ``func``, ``args``, ``keywords`` values.
```

Comment:
", and keywords"
Please wrap line at 79 characters
Please add this below this list:
```
.. versionchanged:: 1.9

    Serialization support for `functools.partial` was added.
```
as well as a sentence in the 1.9 release notes.